src/trackers/TVC.py

```
@@ -147,14 +148,38 @@ async def upload(self, meta, disctype):
 
         await common.edit_torrent(meta, self.tracker, self.source_flag)
         await self.get_tmdb_data(meta)
+        # load MediaInfo and extract audio languages first
+        with open(f"{meta['base_dir']}/tmp/{meta['uuid']}/MediaInfo.json", 'r', encoding='utf-8') as f:
```

Ruff (0.14.2)

Probable insecure usage of temporary file or directory: "/tmp/"

(S108)